### ISSUE TITLE
Graphique en affichage smartphone

### ISSUE THREAD

**Issue by witFGE** [witsa/synapps#453]
L'utilisation d'un acteur Graphique dans une scène affichée sur un smartphone, n'est pas optimisé.
Voici une capture : 
![image](https://user-images.githubusercontent.com/102794133/161242758-15b73206-1669-4541-b347-ce3b3dbd1495.png)

Merci d'avance.

**Comment by manufitoussiwit** [comment]
Effectivement, dans cette orientation, je ne vois pas de solution pour l'instant. Par contre en paysage ça passe bien mieux.
L'idée d'ailleurs serait d'afficher un message lorsqu'on est orienté portrait qui invite à retourner l'écran.
